packages/@ember/-internals/glimmer/lib/helper.ts

```diff
@@ -266,30 +298,6 @@ export const SIMPLE_CLASSIC_HELPER_MANAGER = new SimpleClassicHelperManager();
 
 setHelperManager(() => SIMPLE_CLASSIC_HELPER_MANAGER, Wrapper.prototype);
 
-/**
-  In many cases it is not necessary to use the full `Helper` class.
-  The `helper` method create pure-function helpers without instances.
-  For example:
-
-  ```app/helpers/format-currency.js
-  import { helper } from '@ember/component/helper';
-
-  export default helper(function([cents], {currency}) {
-    return `${currency}${cents * 0.01}`;
-  });
-  ```
-
-  @static
-  @param {Function} helper The helper function
-  @method helper
-  @for @ember/component/helper
-  @public
-  @since 1.13.0
-*/
-export function helper<T, P extends unknown[], N extends Dict<unknown>>(
-  helperFn: HelperFunction<T, P, N>
-): HelperFactory<SimpleHelper<T, P, N>> {
-  return new Wrapper(helperFn);
-}
+export { helper };
```

Comment:
Why the switch to doing this rather than just using `export function helper` as it was originally?

Reply:
No good reason, I'll switch it back.